tests/helpers/test_script.py

```diff
@@ -1080,7 +1081,8 @@ async def test_wait_basic_times_out(hass: HomeAssistant, action_type) -> None:
                         "variables": {"wait": {"remaining": None, "trigger": None}},
                     }
                 ],
-            }
+            },
+            "cancelled",
```

Comment:
The `await script_obj.async_stop()` works now and the state now correctly reflects as `cancelled`